### DISCUSSION TITLE
发现bug

### DISCUSSION THREAD

**Discussion by Burninghamster** [illacloud/illa-builder#1007] [Q&A]
可以给个邀请码吗？？我发现了好多bug
![1677202962964](https://user-images.githubusercontent.com/33534997/221071430-e3e041f8-d694-4e1d-9d13-05f3ea9887f9.png)
2.
![1677203062902](https://user-images.githubusercontent.com/33534997/221071614-f686b114-401a-486b-9751-8fcbf8d56dbb.png)
建议
![1677203393814](https://user-images.githubusercontent.com/33534997/221072271-000d49e1-8461-4cd7-a1a2-21b2c6e0074f.png)
其实就是表格的功能做的还是不够完善，表格这个东西根据实际业务场景来就好了。
![1677203552599](https://user-images.githubusercontent.com/33534997/221072637-2d3da837-deda-460a-8a54-d108d28af816.png)

**Comment by AruSeito** [comment]
您好，请问 ` **oxf@y**t.**fe` 这个是您的邮箱（*为打码内容）吗？前段时间有收到跟你反馈内容一样的邮件。如果是您的邮箱的话麻烦查看邮箱，已经在邮箱内附带了同事的联系方式，邀请码他会发给您的。

**Comment by AruSeito** [comment]
@Burninghamster  您好，您的建议已经反馈给产品同学，BUG 问题近期会修复。然后您看看是否方便带上个人 Github 首页截图，用邮箱联系我(weichen@illasoft.com)一下，我将通过邮箱发送我的个人微信给您，后续也方便沟通交流。
